starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -776,6 +777,20 @@ public static void createVideoMediaCodecBridge(
       }
     }
 
+    if (maxInputSize != -1) {
+      try {
+        mediaFormat.setInteger(MediaFormat.KEY_MAX_INPUT_SIZE, maxInputSize);
+        Log.i(
+            TAG,
+            "KEY_MAX_INPUT_SIZE is "
```

Comment:
nit: We can refine the log to like "Set KEY_MAX_INPUT_SIZE to {maxInputSize} (actual: {mediaFormat.getInteger(android.media.MediaFormat.KEY_MAX_INPUT_SIZE)})."

Reply:
Fixed.